src/components/BooksList.js

```diff
@@ -0,0 +1,43 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const BooksList = props => {
```

Comment:
- [ ] Same concept here (container vs presentational components). `BookList` should be in `src/containers` since it connects to the redux store.